lib/Raven/ErrorHandler.php

```diff
@@ -138,6 +138,8 @@ public function handleFatalError()
                 @$error['message'], 0, @$error['type'],
                 @$error['file'], @$error['line']
             );
+            
+            $this->client->useCompression = $this->client->useCompression && PHP_VERSION_ID > 70000;
```

Comment:
Would be good to comment why PHP version matters here.

Reply:
Well, now we have a regression test with this PR, so removing that check will make the test fail under PHP 5. 

Basically, the reason is that under PHP 7 handling errors is no more a "special place" (thanks to `\Throwabled` probably), so autoloading and other normal stuff work there too.